src/content/news/2021-03-23-real-estate-tech-seminar.md

```
@@ -6,26 +6,45 @@ category: "お知らせ"
 ---
 
 一般社団法人不動産テック協会が主催するセミナーに、代表の宮内が登壇します。
+
+![](/images/news/2021-03-23-real-estate-tech-seminar-1.png)
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)